v2/cf-nodejs-logging-support/src/lib/logger/stacktrace-utils.ts

```diff
@@ -0,0 +1,53 @@
+export class StacktraceUtils {
+
+    private static readonly MAX_STACKTRACE_SIZE = 55 * 1024;
+
+    // check if the given object is an Error with stacktrace using duck typing
+    static isErrorWithStacktrace(obj: any): boolean {
```

Comment:
Nice idea to move the stacktrace functionality to a separate class.
Can we remove the `static` modifiers?